Info/function.json

```diff
@@ -0,0 +1,20 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "route": "v1/info",
```

Comment:
it looks like the info endpoint must be `/info` (without any prefix)
isn't it @pasqualedevita ? this is an issue here as the host.json mandates an `/api` prefix. 
in order to do that we must add an empty prefix in host.json (like [here](https://github.com/pagopa/io-functions-admin/blob/master/host.json#L10)) and rewrite every function.json adding the `/api` prefix in the `route` param.

Reply:
we can change default info endpoint to `/api/v1/info` and then apply same change on fn-admin in `function.json` including full route with prefix.

Reply:
I'd rather apply a fix (with another PR that must be merged before this one) into this repo which is AFAIK the only one that use a prefix inside host.json (the default one = `/api`)

Reply:
ok good!!! I thoughts that rewrite every function.json includes all others repo-functions.

Reply:
ok, then I fix it as io-function-admin in another PR ... also uniform [io-functions-services](https://github.com/pagopa/io-functions-services/blob/master/Info/function.json)?

Reply:
yup cc @francescopersico we're talking here about changin host.json from having a default prefix (api) to having an empty prefix and then set up the full path into every function.json file, do you see any issue with this ?

otherwise it would be impossible to have an `/info ` path without the default prefix

Reply:
since the endpoint is a parameter of the pipeline, it can be /api/v1/info and this pr can be merged afaik cc @balanza